tests/conftest.py

```diff
@@ -6914,3 +6915,36 @@ def setup_logwriter_rbd_workload_factory(request, project_factory, teardown_fact
     )
 
     return logwriter_sts
+
+
+@pytest.fixture()
+def change_noobaa_lifecycle_interval(request):
+    nb_core_pod = get_noobaa_core_pod()
+    interval_changed = False
+
+    def factory(interval):
+        nonlocal interval_changed
+        interval_changed = True
+        change_expiration_query_interval(new_interval=interval)
+
+    def finalizer():
+        if interval_changed:
+            params = (
+                '[{"op": "test", "path": "/spec/template/spec/containers/0/env/20/name",'
+                '"value": "CONFIG_JS_LIFECYCLE_INTERVAL"},'
+                '{"op": "remove", "path": "/spec/template/spec/containers/0/env/20"}]'
```

Comment:
I suggest using the existing `add_env_vars_to_noobaa_core_class` fixture which allows to do the same change (and also takes care of the cleanup). This is how I'm using it in #8210 (still not merged, unfortunately):

```
    @pytest.fixture(scope="class", autouse=True)
    def reduce_expiration_interval(self, add_env_vars_to_noobaa_core_class):
        """
        Reduce the interval in which the lifecycle background worker is running

        """
        new_interval_in_miliseconds = 60 * 1000
        add_env_vars_to_noobaa_core_class(
            [(constants.LIFECYCLE_INTERVAL_PARAM, new_interval_in_miliseconds)]
        )
```